sound/effect.h

```diff
@@ -175,6 +183,11 @@ class Effect {
     paired_ = false;
 #ifdef KILL_OLD_PLAYERS
     killable_ = false;
+#endif
+#ifdef NO_REPEAT_RANDOM
```

Comment:
If we're going to have this code (which I'm not entirely convinced it is a good idea.) then we should use a new define to enable it. Maybe the NO_REPEAT_RANDOM_BUFFER_SIZE_BITS define?